packages/shared/src/components/squads/moderation/SquadModerationList.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactElement } from 'react';
+import { Button } from '../../buttons/Button';
+import { ButtonVariant, ButtonSize } from '../../buttons/common';
+import { VIcon } from '../../icons';
+import { useSquadPostModeration } from '../../../hooks/squads/useSquadPostModeration';
+import { IconSize } from '../../Icon';
+import {
+  Typography,
+  TypographyColor,
+  TypographyType,
+} from '../../typography/Typography';
+import { useSquadPendingPosts } from '../../../hooks/squads/useSquadPendingPosts';
+import { SquadModerationItem } from './SquadModerationItem';
+
+export function SquadModerationList(): ReactElement {
+  const { onApprove, onReject, isLoading } = useSquadPostModeration();
+  const { data, isFetched } = useSquadPendingPosts();
+
+  if (!data?.length) {
+    if (!isFetched) {
+      return (
+        <div className="flex flex-col gap-4 p-6">
+          <span className="typo-title3">Loading...</span>
```

Comment:
We don't have yet a skeleton, but maybe we can come up with a simple one.

Reply:
Probably good idea indeed. :) 
Just basic one based on the elements is fine

Reply:
Should look like below 👀 

![Screenshot 2024-10-09 at 8 37 57 PM](https://github.com/user-attachments/assets/d5fb05e2-3c22-4bd9-8d25-de3a184028dd)